### ISSUE TITLE
False-positive: "Objects are always passed as reference" on nullable scalars

### ISSUE THREAD

**Issue by GrandLTU** [kalessil/phpinspectionsea#505]
![Example](https://image.prntscr.com/image/0GK7Qg1hSpCppi0EtrZ7qg.png)

**Comment by kalessil** [comment]
Thank you for reporting. Looks like a bug, but I need more information to find the root cause.
@GrandLTU, can you share the file, or construct a code fragment where the same is reproducible (drop body statements, all extra parameters, other methods/field)?

**Comment by GrandLTU** [comment]
sure, just a simple function triggers it:
```php
<?php

function foo(?string &$bar)
{
}
```

**Comment by kalessil** [comment]
Fixed, the new release will happen ~exnd of the next week!
